# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-disable-rust-profiler.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-133.0.6943.60/build/rust/std/BUILD.gn.vanilla	2025-02-12 17:10:04.590740350 +0100
+++ chromium-133.0.6943.60/build/rust/std/BUILD.gn	2025-02-12 17:10:35.555770653 +0100
@@ -115,7 +115,6 @@
   # don't need to pass to the C++ linker because they're used for specialized
   # purposes.
   skip_stdlib_files = [
-    "profiler_builtins",
     "rustc_std_workspace_alloc",
     "rustc_std_workspace_core",
     "rustc_std_workspace_std",
